examples/all-clusters-app/esp32/main/CMakeLists.txt

```diff
@@ -59,7 +61,31 @@ idf_component_register(PRIV_INCLUDE_DIRS
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ias-zone-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/relative-humidity-measurement-server"
                       #${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ias-zone-client
-                      PRIV_REQUIRES chip QRCode tft spidriver bt screen-framework)
+)
+
+if(${CONFIG_DEVICE_TYPE_ESP32_C3_DEVKITM})
+set(PRIV_INCLUDE_DIRS_GROUPS ${PRIV_INCLUDE_DIRS_GROUPS}
```

Comment:
The same as the following `if` block. Can be merged with all the following `if` blocks.

Reply:
Changed